src/store/useLoggerStore.ts

```diff
@@ -21,7 +21,12 @@ const store: StateCreator<LoggerStore> = (set) => ({
 		set(() => ({
 			logs: [],
 		})),
+	switchLogger: (enabled) =>
```

Comment:
maybe toggleLogger?
The switch is more like a change; here we're turning it on or off, which is a toggle.
Pick one of: `toggleLogger` or `showHideLogger` or `toggleLoggerVisibility`
Also, the param name should be `isEnabled` or `isVisible` or `newState` or `displayState`

Reply:
Obviously, isLoggerEnabled is clearer, `isEnabled `is shorter, but it is not clear what it is. What do you think?